io.github.syntalos.yaml

```diff
@@ -0,0 +1,244 @@
+id: io.github.syntalos
+runtime: org.kde.Platform
+runtime-version: 5.15-23.08
+sdk: org.kde.Sdk
+base: com.riverbankcomputing.PyQt.BaseApp
+base-version: 5.15-23.08
+separate-locales: false
+
+command: syntalos
+rename-icon: syntalos
+rename-desktop-file: syntalos.desktop
+
+finish-args:
+  - --share=network
+  - --share=ipc
+  - --socket=x11
+  - --device=all
+  - --filesystem=host
```

Comment:
why is host needed?

Reply:
People use this software in environments where they will want to store data in locations like `/mnt`, `/srv` or even custom other paths depending on what policy the research organization has for live data storage.
We have no idea where data will be saved in advance, and given that the software is already very weakly sandboxed, just allowing most locations will result in the fewest complaints from users when they use the Flatpak bundle over a native package  :-)